### PR TITLE
448 allow custom conversions on stream config

### LINKED ISSUE #448: Feature: RPM to flow/time unit conversion - allow customConversions in the stream config
To support unit conversion from rpm to ml/min we need a way to store the user input for the ratio between rotation and ml. This should go into a new field on the stream configuration named `customConversions` and should be something like this: 

```js
customConversions: [ { to: 'ml/min', ratio: 0.4 }]
```

adjust the Joi schema for updateDevice to allow this new property on the stream configuration and add tests for it.

### PR DESCRIPTION
**Pull request by vdiezel**
closes #448

### CODE REVIEW

File: services/device/test/device/update.test.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ describe('Device Class', () => {
                 slope: 1.2,
               },
               eventTriggers: [],
+              customConversions: [{ ratio: 0.4, to: 'ml/min' }],
```
Comment (ConorTighe1995) [inline services/device/test/device/update.test.js:85]: This may be a stupid question but why is it 0.4 here?

Comment (vdiezel) [reply services/device/test/device/update.test.js:85]: just a random number a user might have put in - has no meaning at all

**Comment by vdiezel** [comment]
> this is looking good
> 
> Might also be time to make a device stream SCHEMA. Only this could also work for now, but I think we may want our data to be verified outside of routes themselves

I agree. I will postpone this to when we implement the device model!

**Comment by vdiezel** [comment]
replaced by #460

**Pull request closed without merging**